tests/task_t_1.spec.js

```diff
@@ -48,3 +44,49 @@ test('T4: Check that URL shortening feature works', async ({ page }) => {
   const newPageURL = await page.url();
   expect(newPageURL).toBe("https://www.wikipedia.org/");
 });
+
+test('T5: Check that intitle functionality works', async ({ page }) => {
+  const searchInputField = page.locator('#search_form_input_homepage');
+  await searchInputField.fill('intitle:panda');
+  await page.click('#search_button_homepage');
+
```

Comment:
` await page.waitForNavigation();` - suggested addition here, to make test more stable

Reply:
fixed